scraper/tfs/__init__.py

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+import logging
+import os
+
+from vsts.vss_connection import VssConnection
+from msrest.authentication import BasicAuthentication
+from scraper.tfs.models import TFSProject
+
+logger = logging.getLogger(__name__)
+
+HARD_CODED_TOP = 10000
+
+
+def get_projects_metadata(baseurl, token):
```

Comment:
Is this method needed if all of the functionality is in `populate_tfs_projects()` ?

Reply:
Agree. populate_tfs_projects is now gone and only get_projects_metadata remains.